src/main/java/net/galgus/flink/streaming/connectors/http/HTTPSink.java

```diff
@@ -5,14 +5,16 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import javax.net.ssl.HttpsURLConnection;
+import javax.net.ssl.*;
```

Comment:
switch back to a specific import. wild imports are prohibited by the style guide.